ESP_ATMod/ESP_ATMod.h

```diff
@@ -84,7 +84,8 @@ extern bool gsEchoEnabled;		// command ATE
 extern uint8_t gsCipMux;		// command AT+CIPMUX
 extern uint8_t gsCipdInfo;		// command AT+CIPDINFO
 extern uint8_t gsCwDhcp;		// command AT+CWDHCP
-extern bool gsFlag_Connecting;	// Connecting in progress (CWJAP) - other commands ignored
+extern bool gsFlag_Connecting;	// Connecting in progress

```

Comment:
At a first glance I suspect, the `gsFlag_Connecting` and `gsFlag_Busy` are synonyms. If it is so, I'd be in favor to use `gsFlag_Busy` only.

Reply:
No they are not, `gsFlag_Connecting` is your function to check for a connecting. `gsFlagBusy`  is simply just letting the user/library know that the OS is busy. Which can be with connecting as used previously, but can also be with something else/other command. Like the CWLAP command now.